### DISCUSSION TITLE
New Projects Unable to Assign

### DISCUSSION THREAD

**Discussion by xipasduarte** [community/community#11013] [Projects and Issues]
When editing an item on a project in Kanban view the "Assignees" doesn't allow for any interactions.

I tried Firefox and Chromium and the result is the same. the is no possible interaction for the assignees.
![Screenshot 2022-02-04 at 14 25 04](https://user-images.githubusercontent.com/123991/152547950-1b947604-7f4c-4e3f-93f8-2389ae1ddb60.png)

**Comment by ecbrodie** [comment]
It appears that assignees can only be set for non-issue items via the Table View. For the KanBan view, it's a completely un-settable field.

Or, you are forced to convert these items to issues, at which case, setting an assignee is just done via the Issue's view itself. But once again, it's cleaner to set the assignee via the project's Table View, via a dropdown.

So, TLDR, there is work that still needs to be done to add functionality to the KanBan View.

**Reply by xipasduarte** [reply]
Yes, only the Kanban view is affected. I've updated the description to include this.
